DOL.WHD.Section14c.Web/src/modules/pages/landingPageController.js

```diff
@@ -4,20 +4,65 @@ module.exports = function(ngModule) {
   ngModule.controller('landingPageController', function(
     $scope,
     stateService,
+    autoSaveService,
+    apiService,
     $location
   ) {
     'ngInject';
     'use strict';
 
-    $scope.stateService = stateService;
+    apiService.userInfo(stateService.access_token).then(function(result) {
+      $scope.organizations = result.data.organizations;
+    });
+
 
     $scope.changePassword = function() {
       $location.path('/changePassword');
     };
 
-    $scope.navToApplication = function() {
+    $scope.onApplicationClick = function(index) {
+
+      stateService.employerId = $scope.organizations[index].employer.id;
+      stateService.applicationId = $scope.organizations[index].applicationId;
+      stateService.ein = $scope.organizations[index].ein;
+      if($scope.organizations[index].applicationStatus.name === "New") {
+         stateService.saveNewApplication().then(
+          function() {
+            // start auto-save
+            if (stateService.ein) {
+              autoSaveService.start();
+              $location.path('/section/assurances');
+            }
+
+          },
+          function() {
+          }
```

Comment:
Should something happen if saving the new application fails for some reason?

Reply:
addressed. I implemented error feedback for application Load